api-reference/v1.0/api/domain_delete.md

```diff
@@ -0,0 +1,65 @@
+# Delete domain
```

Comment:
I wonder whether we want to link to this topic:  api-reference/v1.0/api/domain_list_domainnamereferences.md.  Attempts to delete will fail if there are any resources/objects still dependent on the domain.  You can find all those dependent resources with the API call (in the link).